python/foglamp/services/core/api/auth.py

```diff
@@ -7,15 +7,20 @@
 """ auth routes """
 
 import re
+import os
+import json
 from collections import OrderedDict
 
 from aiohttp import web
 from foglamp.services.core.user_model import User
 from foglamp.common.web.middleware import has_permission
 from foglamp.common import logger
+from foglamp.common.web.tls_helper import BaseCert, VerificationError
+from foglamp.common.configuration_manager import ConfigurationManager
```

Comment:
Should not we use user_model for db interaction? its already imported there. We can call the verify_certificate(cert, certs_dir): defined in user_model